### DISCUSSION TITLE
Customise output class

### DISCUSSION THREAD

**Discussion by simtabi** [BetaHuhn/drkmd.js#67] [Ideas]
Is there a way to control the current behavior in how the following is outputted?

`theme-dark ` it would be best for a user to be able to dictate on what class is outputted, and in this case, in the plugin options, one should be able to do the following:

```
const options = {
  ...
  outputClass: 'dark-theme', // or any other according to your layout design
}
```

Thanks

**Comment by BetaHuhn** [comment]
This is not currently possible, but I like the idea!

It should be a simple change, do you feel comfortable creating a PR for it yourself?
